survey-runner-queue/terraform.tfvars.example

```diff
@@ -19,4 +19,7 @@ application_cidrs=["10.30.20.48/28"]
 sdx_cidrs=["10.99.0.1/32"]
 rsyslogd_server_ip="10.99.0.2"
 logserver_cidr="10.99.0.3/32"
-audit_cidr="10.99.0.4/32"
\ No newline at end of file
+audit_cidr="10.99.0.4/32"
+
+# Should be true in developer environment to avoid build up of volumes
+delete_on_termination="false"
```

Comment:
The other values in the example files are set to sensible defaults for developers to get up and running quickly, so it would make sense for this to be `true` rather than `false`. Also I think the value should be unquoted as it's a boolean.